### ISSUE TITLE
Wrong flag for Cocos (Keeling) Islands 

### ISSUE THREAD

**Issue by coulisse** [coulisse/spiderweb#23]
**Describe the bug**
The flag of Cocos (Keeling) Islands is not correct

**To Reproduce**
Search for VK9CM and see the flag is orizontal blue / white / red  (COSTA RICA) - Iso Code: **CR**

**Expected behavior**
The right flag is green and yellow - Iso Code: **CC**

**Screenshots**
Error:
![image](https://user-images.githubusercontent.com/25729924/199462573-d70e919a-a5fc-4b6a-aaf7-2ace9c93ea34.png)

Expected:
![image](https://user-images.githubusercontent.com/25729924/199462674-8698275b-d71b-4464-afe8-108bb5a773ae.png)

**Comment by coulisse** [comment]
Is necessary to modify the country file spiderweb/cfg/country.json, changing:
```
{ "WPX": "169", "ISO": "cr", "desc": "Cocos Island" },
{ "WPX": "169", "ISO": "cr", "desc": "Cocos (Keeling) Islands" },
```

in
```
{ "WPX": "169", "ISO": "cc", "desc": "Cocos Island" },
{ "WPX": "169", "ISO": "cc", "desc": "Cocos (Keeling) Islands" },
```

**Comment by coulisse** [comment]
Closed with release 2.4
